pkg/clients/k8s/client.go

```diff
@@ -88,19 +86,27 @@ func New(ctx context.Context, rest *rest.Config) (*Clients, error) {
 	if err != nil {
 		return nil, err
 	}
-	opts := &generic.FactoryOptions{
-		SharedControllerFactory: factory,
+	settingGVR := schema.GroupVersionResource{Group: "management.cattle.io", Version: "v3", Resource: "settings"}
+	settingKind := "Setting"
+	settingController := factory.ForResourceKind(settingGVR, settingKind, false)
+	err = settingController.Start(context.Background(), 1)
+	if err != nil {
+		return nil, fmt.Errorf("error when starting setting controller %w", err)
 	}
-	mgmt, err := management.NewFactoryFromConfigWithOptions(rest, opts)
+
+	notificationGVR := schema.GroupVersionResource{Group: "management.cattle.io", Version: "v3", Resource: "rancherusernotifications"}
+	notificationKind := "RancherUserNotification"
+	notificationController := factory.ForResourceKind(notificationGVR, notificationKind, false)
+	err = notificationController.Start(context.Background(), 1)
 	if err != nil {
-		return nil, err
+		return nil, fmt.Errorf("error when starting notification controller %w", err)
```

Comment:
Interesting, I didn't know about the `%w` verb, cool!

Reply:
Yeah @KevinJoiner showed me that - lets the calling function do matching on the error type easier.